t/ui/18-tests-details.t

```diff
@@ -57,7 +58,8 @@ t::ui::PhantomTest::wait_for_ajax;
 ok($driver->find_element('#preview_container_out', 'css')->is_displayed(), "preview window opens on click");
 like($driver->find_element('#preview_container_in', 'css')->get_text(), qr/wait_serial expected/, "Preview text with wait_serial output shown");
 $driver->find_element('[title="wait_serial"]', 'css')->click();
-t::ui::PhantomTest::wait_for_ajax;
+# wait for animation to finish
+sleep .2;
```

Comment:
My comment was meant to be sarcastic as in: "Oh, why 0.2 seconds, why not 0.1, why not even 1000 seconds?" The reasoning for 0.2 seconds is certainly not obvious. I am sure you have good reasons for choosing 0.2 seconds. "t::ui::PhantomTest::wait_for_ajax;" sounds much nicer because at least from the tests perspective it states what should really be done here and leaves everything else to PhantomTest itself, e.g. if this one implements "wait_for_ajax" internally even with a simple sleep, too. I tested this code initially without needing any waiting so I am not sure what is different with my setup to yours or travis but should we able to find a proper solution like "wait_for_ajax"? How can I make "wait_for_ajax" fail to reproduce any issue you might have encountered?